forge.lic

```diff
@@ -75,7 +75,7 @@ class Forge
 
   def temper
     unless checkleft =~ /#{@item}/i || checkright =~ /#{@item}/i
-      echo('please hold the item to temper')
+      echo('***Please hold the item to temper.***')
```

Comment:
There should be an effort (once this is merged) to make "smith" style scripts for sew/shape/carve and do similar refactoring there.